zeebe/clients/java/src/main/java/io/camunda/zeebe/client/impl/http/HttpClient.java

```diff
@@ -95,16 +94,37 @@ public <HttpT, RespT> void get(
       final Class<HttpT> responseType,
       final JsonResponseTransformer<HttpT, RespT> transformer,
       final HttpZeebeFuture<RespT> result) {
+    sendRequest(path, null, requestConfig, responseType, transformer, result);
+  }
+
+  public <HttpT, RespT> void post(
+      final String path,
+      final String body,
+      final RequestConfig requestConfig,
+      final HttpZeebeFuture<RespT> result) {
+    sendRequest(path, body, requestConfig, Void.class, r -> null, result);
```

Comment:
:thought_balloon: I guess we'll refactor this method once we implement HTTP POST endpoints that return some content in the response.

Reply:
Yep, but we don't have any yet, so 🤷